lib/taskana-core/src/test/resources/taskana.properties

```diff
@@ -6,7 +6,7 @@ taskana.roles.monitor=john|teamlead_2 | monitor
 taskana.domains= Domain_A  ,  DOMAIN_B
 
 taskana.classification.types= TASK , document
-taskana.classification.categories= EXTERNAL , manual, autoMAtic  ,Process
+taskana.classification.categories= TASK: EXTERNAL , manual, autoMAtic, Process; document: Default;
```

Comment:
I would prefer the following configuration:
taskana.classification.categories.TASK= EXTERNAL, ...
taskana.classification.categories.DOCUMENT= EXTERNAL, ...
What do you think?

Reply:
Your way is scalable. Thank you for your advice